core/tx_pool.go

```diff
@@ -884,13 +899,17 @@ func (pool *TxPool) addTxsLocked(txs []*types.Transaction, local bool) []error {
 	// Add the batch of transaction, tracking the accepted ones
 	dirty := make(map[common.Address]struct{})
 	errs := make([]error, len(txs))
+	var erroredTxns []types.RPCTransactionError
```

Comment:
prefer short declaration, errorTxns := []foo{}

Reply:
fix in latest